spec/carthage_cache/archiver_spec.rb

```diff
@@ -10,7 +10,7 @@
   describe "#archive" do
 
     it "creates a zip file with the content of the project's 'Carthage/Build' directory" do
-      expected_command = "cd #{build_directory} && zip -r -X #{archive_path} iOS Mac tvOS watchOS > /dev/null"
+      expected_command = "cd #{build_directory} && zip -r -X #{archive_path}  > /dev/null"
```

Comment:
It was breaking the spec, but the change you outlined below should hopefully fix it.